src/worker/api/index.ts

```diff
@@ -91,6 +94,24 @@ import {
 } from "../core/season/awards";
 import { getScore } from "../core/player/checkJerseyNumberRetirement";
 import type { NewLeagueTeam } from "../../ui/views/NewLeague/types";
+import * as filter_functions from "../core/trade/filterFunctions";
+
+export type OfferType = {
+	tid: number;
+	abbrev: string;
+	region: string;
+	name: string;
+	strategy: string;
+	won: number;
+	lost: number;
+	tied: number;
```

Comment:
Should also have `otl: number;` from https://github.com/mchvn/gm-games/commit/07d2a056ac3bb6f838419ac163feae24d57aa478